packages/data-broker/src/__tests__/DataBroker/DataBroker.test.js

```diff
@@ -190,8 +197,8 @@ describe('DataBroker', () => {
     });
 
     describe('events', () => {
-      const assertServicePulledEventsOnce = (service, dataElements) =>
-        expect(service.pull).toHaveBeenCalledOnceWith(dataElements, 'dataGroup', options);
+      const assertServicePulledEventsOnce = (service, dataElements, optionsOverride = options) =>
```

Comment:
I can't see how this and the similar case above are actually using `optionsOverride`. In every instance it seems to just pass in the same default options?

Reply:
Good catch, I had changed these tests cos previously I was overriding the options at points, but then removed that code but forgot to update here! Will fix

Reply:
One down, one to go ;-)

Reply:
Ahh woops 🤦‍♂️ Thank u!